modules/mh_phab.py

```diff
@@ -63,7 +63,7 @@ def searchphab(bot, channel, task=1):
         priority = result.get("fields").get("priority").get("name")
         status = result.get("fields").get("status").get("name")
         output = 'https://phabricator.miraheze.org/T{0} - '.format(str(result["id"]))
-        output = '{0}\x02{1}\x02, authored by \x02{2}\x02, assigned to \x02{3}\x02, Priority: \x02{4}\x02, Status: \x02{5}\x02'.format(output, str(result.get("fields").get("name")), author, str(owner), priority, status)
+        output = '{0}{1}, authored by {2}, assigned to {3}, Priority: {4}, Status: {5}'.format(output, str(result.get("fields").get("name")), author, str(owner), priority, status)
```

Comment:
These were added to make it bold, can we have them back?

Reply:
Ahh... in the future, please, please, please document such things.